src/main/src/main/java/org/geoserver/catalog/LayerGroupInfo.java

```diff
@@ -88,6 +88,19 @@ public Integer getCode() {
     void setMode( Mode mode );    
 
     /**
+     * Get whether the layer group is forced to be not queryable and hence can not be subject of a GetFeatureInfo request.
+     * <p>
+     * Default is FALSE
+     * </p>
+     */
+    Boolean isNotQueryableEnabled();
```

Comment:
Make this isQueryable like in LayerInfo. Actually, you could just push up the isQueryable/setQueryable methods up from LayerInfo to PublishedInfo, which is the base interface for LayerInfo and PublishedInfo